fmn/consumer/backends/mail.py

```diff
@@ -148,8 +148,17 @@ def _format_line(msg):
 
         n = len(queued_messages)
         subject = u"Fedora Notifications Digest (%i updates)" % n
+        summary = u"Digest summary:\n"
```

Comment:
A request:  Can we only prepend the summary if ``verbose`` is False?  It seems like the only substantive difference between the summary and the actual body is the possible presence of ``long_form`` content.